apps/docs/src/app/core/documentation/core-documentation-data.ts

```diff
@@ -112,7 +112,8 @@ export const utilities: SectionInterfaceContent[] = [
     { url: 'core/initial-focus', name: 'Initial Focus' },
     { url: 'core/scroll-spy', name: 'Scroll Spy' },
     { url: 'core/theme-switcher', name: 'Theme Switcher API' },
-    { url: 'core/theming', name: 'Theming' }
+    { url: 'core/theming', name: 'Theming' },
+    { url: 'core/content-density-2', name: 'Content Density 2' }
```

Comment:
not sure about `-2` naming at all because we have to rename it in future. maybe we can rename the previous docs folder to `content-density-old` or something like that? and the new one will be just a `content-density`. then, somewhen in future we will just remove the old one.